Sources/Cells/SelectableTableViewCell.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+open class SelectableTableViewCell: UITableViewCell {
+
+    // MARK: - Init
+
+    public override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setup()
+    }
+
+    required public init?(coder aDecoder: NSCoder) { fatalError() }
+
+    // MARK: - Setup
+
+    private func setup() {
+        let selectedBackgroundView = UIView()
```

Comment:
How about moving this function to extension:

```swift
extension UITableViewCell {
    func setDefaultSelectedBackgound() {
        let selectedBackgroundView = UIView()
        selectedBackgroundView.backgroundColor = .defaultCellSelectedBackgroundColor
	self.selectedBackgroundView = selectedBackgroundView
    }
}
```

`BasicTableViewCell` would call this function during its setup, and we can also call it from any other table view cell subclass.